### DISCUSSION TITLE
Revalidate/Preview on edge api routes

### DISCUSSION THREAD

**Discussion by alex289** [vercel/next.js#40320] [Help]
Hey, is it possible to use ISR or the preview mode with the edge runtime. Since the response object is not passed as a parameter I was wondering how this works

**Comment by petr166** [comment]
Was this closed because of an available solution? If so could you point me to it? 😁

**Reply by alex289** [reply]
I didnt get an solution :(

The new app router makes this kinda obsolete I guess
